addons/sale_timesheet/tests/test_sale_timesheet.py

```diff
@@ -969,6 +970,36 @@ def test_onchange_uom_service_product(self):
                 product_form.detailed_type = 'consu'
                 self.assertEqual(product_form.uom_id.id, uom_kg.id)
 
+    def test_non_consolidated_billing_service_timesheet(self):
```

Comment:
Could use a docstring or some comments to explain what you're testing/why you're testing it